package.json

```diff
@@ -71,4 +71,4 @@
     "stylelint": "^9.3.0",
     "stylelint-config-standard": "^18.2.0"
   }
-}
+}
```

Comment:
Add newline, please. It will be easier to set your editor to automatically add it for you